fleetspeak/src/server/https/streaming_message_server.go

```diff
@@ -120,8 +120,12 @@ func (s streamingMessageServer) ServeHTTP(res http.ResponseWriter, req *http.Req
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	info, moreMsgs, err := s.initialPoll(ctx, addrFromString(req.RemoteAddr), cert.PublicKey, fullRes, body)
+	addr := addrFromString(req.RemoteAddr)
+	info, moreMsgs, err := s.initialPoll(ctx, addr, cert.PublicKey, fullRes, body)
 	if err != nil || info == nil {
+		if err {
```

Comment:
I'm surprised this line compiles.

Reply:
Fun fact - it doesn't. I wonder how are presubmit checks are passing in this case? Will take a closer look. Updated the code in the meantime.

Reply:
For some reason travis is not running.